src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -272,7 +275,8 @@ private void WriteValue(Value value, bool isFirst = true, bool valueInList = fal
 
         private void WriteValue(string value)
         {
-            _writer.Write($"\"{value}\"");
+            _writer.Flush();
+            s_jsonStringSerializer.WriteObject(_writer.BaseStream, value);
```

Comment:
using writer with the serializer might be cheaper. WriteObject seems create writer on every call.